### ISSUE TITLE
Accessability

### ISSUE THREAD

**Issue by bias005** [datahub-project/datahub#8962]
**Describe the bug**
In Germany, there will be a new law on accessibility from 2025, which is why many companies are already implementing the guidelines. Unfortunately, Datahubproject does not comply with them, which is why we cannot use it. Required would be, for example, mouse-free navigation and reasonable usability with a screen reader.

Are there any plans to work on the accessibility of datahubproject or should we do this ourselves?
